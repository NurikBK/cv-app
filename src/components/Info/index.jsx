import React from 'react';
import { StyledInfo } from '../styles/Info.styled';

const Info = ({ text }) => {
  return (
    <StyledInfo>
      <p>{text}</p>
    </StyledInfo>
  );
};

export default Info;
