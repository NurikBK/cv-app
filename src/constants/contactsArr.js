import { faPhone, faEnvelope } from '@fortawesome/free-solid-svg-icons';
import {
  faTwitter,
  faFacebookF,
  faSkype,
} from '@fortawesome/free-brands-svg-icons';

export const contacts = [
  {
    icon: faPhone,
    title: '+7 708 693 23 41',
    info: '',
    link: 'tel:+7 708 693 23 41',
  },
  {
    icon: faEnvelope,
    title: 'bizhannk@gmail.com',
    info: '',
    link: 'mailto:bizhannk@gmail.com',
  },
  {
    icon: faTwitter,
    title: 'Twitter',
    info: '@Nurik',
    link: 'https://twitter.com/wordpress',
  },
  {
    icon: faFacebookF,
    title: 'Facebook',
    info: 'Nurzhan Bizhan',
    link: 'https://www.facebook.com/facebook',
  },
  {
    icon: faSkype,
    title: 'Skype',
    info: 'Nurik',
    link: 'skype:nurik',
  },
];
