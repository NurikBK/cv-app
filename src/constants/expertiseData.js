export const expertiseData = [
  {
    date: '2013-2023',
    info: {
      company: 'Asia Gas Pipeline',
      job: 'Front-end developer / php programmer',
      description: `Translate written and spoken content accurately and efficiently from Chinese to English, English to Chinese, Russian to English, and English to Russian
        Proofread and edit translations to ensure accuracy and clarity of the content
        Research industry-specific terminology to ensure that translations are accurate and appropriate
        Use translation software and tools to assist in the translation process
        Collaborate with other translators or editors to ensure consistency and accuracy across translations
        Maintain confidentiality of translated content and client information
        Communicate with clients to understand their needs and preferences and ensure that the translated content meets their expectations
        Meet project deadlines and manage time effectively`,
    },
  },
  {
    date: '2023 feb - 2023 may',
    info: {
      company: 'Bank Center Credit',
      job: 'Junior Web developer',
      description: `Creating and managing content for social media platforms such as Facebook, Twitter, and Instagram.

        Writing blog posts and articles for the company website.
        
        Conducting research and gathering information on industry-related topics to inform content creation.
        
        Collaborating with the marketing team to create and execute social media and content marketing strategies.
        
        Managing social media accounts by monitoring engagement and responding to user comments and messages.
        
        Creating and sending newsletters and email marketing campaigns to subscribers.
        
        Conducting website updates and maintenance tasks as needed.
        
        Managing Google Analytics and other marketing performance tracking tools to measure success of content and social media campaigns.
      `,
    },
  },
];
