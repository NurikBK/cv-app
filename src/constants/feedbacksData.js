export const feedbackData = [
  {
    feedback: `I recently worked with Nurzhan on a web development project, and I was thoroughly impressed with their skills and knowledge. They were able to take my design and turn it into a fully functional and responsive website using React and Material UI. Their attention to detail and ability to problem-solve made the project run smoothly, and the end result exceeded my expectations. I highly recommend Nurzhan for any web development projects.`,
    reporter: {
      photoUrl: 'https://i.pravatar.cc/150?img=3',
      name: 'John Doe',
      citeUrl: 'https://www.citeexample.com',
    },
  },
  {
    feedback: `I had the pleasure of working with Nurzhan on a complex web development project that involved NextJS and SCSS. They were able to navigate through the challenges of the project with ease, and their ability to collaborate with the rest of the team made the project a success. Their understanding of the latest web development technologies and their attention to detail resulted in a website that is not only visually stunning but also highly functional. I highly recommend Nurzhan for any web development projects`,
    reporter: {
      photoUrl: 'https://i.pravatar.cc/150?img=10',
      name: 'Jane Rober',
      citeUrl: 'https://www.citeexample.com',
    },
  },
];
