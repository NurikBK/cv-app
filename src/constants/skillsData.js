export const skillsData = [
  { id: 1, name: 'CSS', range: '60' },
  { id: 2, name: 'HMTL', range: '80' },
];
