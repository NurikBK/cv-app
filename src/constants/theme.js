export const theme = {
  colors: {
    accent: '#26c17e',
    secondary: '#fff',
    dark: '#667081',
    darkBg: '#222935',
  },
};
