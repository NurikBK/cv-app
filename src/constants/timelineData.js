export const timelineData = [
  {
    date: '2011',
    title: 'Dalian University Of Foreign Languages',
    text: ` I attended Dalian University of Foreign Languages (DLUFL) in Liaoning, China, where I pursued a Bachelor of Arts in Chinese Literature. Throughout my four years of study, I was exposed to a wide range of literature, from classic works to contemporary pieces. In addition to studying literature in Chinese, I also had the opportunity to learn about Chinese literature and culture through elective courses. Outside of the classroom, I participated in various extracurricular activities, including a literature club. These experiences broadened my perspective and deepened my appreciation for the power of language and storytelling.`,
  },
  {
    date: '2012',
    title: 'Master of Business Administration',
    text: `I earned my Master of Business Administration (MBA) degree from Alamty University of Bussnes in the Kazakhstan. The program provided me with a comprehensive understanding of business concepts, such as marketing, finance, accounting, and operations management. I also had the opportunity to specialize in entrepreneurship and innovation, which allowed me to develop skills in identifying and pursuing new business opportunities. Throughout the program, I worked on various group projects, which improved my collaboration and leadership skills. The MBA program also provided me with a strong professional network that I can tap into for career opportunities.`,
  },
  {
    date: '2023',
    title: 'EPAM',
    text: `EPAM Systems, Inc. is an American company that specializes in software engineering services, digital platform engineering, and digital product design, operating out of Newtown, Pennsylvania. I finished the EPAM web developer course, where I learned about HTML, CSS, JS, and React 'During my studies, I gained a deep understanding of computer architecture, algorithms, data structures, programming languages such as HTML, CSS, JavaSctripts and React. I also had the opportunity to work on various projects, including a web app for tracking fitness goals and a web application for online shopping. These experiences honed my skills in problem-solving, collaboration, and communication. Overall, my degree program provided me with a strong foundation in web development and prepared me for a career in web development.`,
  },
];
