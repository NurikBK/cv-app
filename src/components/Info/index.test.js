import React from 'react';
import { render, screen } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';
import { theme } from '../../constants/theme';
import Info from '.';

describe('Info component', () => {
  it('renders info with text prop', () => {
    render(
      <ThemeProvider theme={theme}>
        <Info text="back" />
      </ThemeProvider>
    );
    const text = screen.getByText(/back/i);
    expect(text).toBeInTheDocument();
  });
});
