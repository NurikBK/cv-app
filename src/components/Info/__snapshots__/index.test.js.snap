// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info component renders info with text prop 1`] = `
<p>
  back
</p>
`;
